my_cv/css/style.css

```diff
@@ -0,0 +1,92 @@
+*{
+    text-decoration: none;
+    list-style:circle;
+
+}
+
+body {
+    max-width: 1024px;
+    margin: auto;
+    padding: 5px;
+    display: block;
+}
+
+h1{
```

Comment:
When you add CSS directly on tags, your markup becomes unscalable. Pls take a look - https://frontstuff.io/you-need-to-stop-targeting-tags-in-css